dart/tool/service.md

```diff
@@ -1309,8 +1309,7 @@ class Class extends Object {
   bool const;
 
   // The library which contains this class.
-  // TODO: This should be @Library, but the VM can return @Instance objects here.
```

Comment:
🍾 

This is the last diff from the dart-lang/sdk spec?

Reply:
Yep!